api/tests/routes/public/individual_offers/v1/post_product_test.py

```diff
@@ -565,7 +565,7 @@ def test_not_allowed_categories(self, client):
                 "unexpected value; permitted: 'VISITE_VIRTUELLE'",
                 "unexpected value; permitted: 'VOD'",
             ],
-            "categoryRelatedFields.ean": ["field required", "field required", "field required"],
+            "categoryRelatedFields.ean": ["field required", "field required", "field required, field required"],
```

Comment:
Typo et il en manque 1: 

```suggestion
            "categoryRelatedFields.ean": ["field required", "field required", "field required", "field required", "field required"],
```